pgpkey/pgpkey_test.go

```diff
@@ -622,6 +617,15 @@ func makeKeyWithSubkeys(t *testing.T, subkeyConfigs []subkeyConfig, now time.Tim
 	return pgpKey, nil
 }
 
+func assertSubkeyValiditity(subkey openpgp.Subkey, expectedIsValid bool, now time.Time, t *testing.T) {
```

Comment:
Nitpick here, but it's not 100% clear that "assert validity" means "assert *is valid*"

Reply:
Ohhh... it actually takes `expectedIsValid`, my bad